runtime/src/native_loader.rs

```diff
@@ -81,7 +88,12 @@ pub fn entrypoint(
                             return Err(InstructionError::GenericError);
                         }
                     };
-                return entrypoint(program_id, params, ix_data, tick_height);
+                let ret = entrypoint(program_id, params, ix_data, tick_height);
```

Comment:
Can we consolidate to call the entrypoint in only one place?

Reply:
ok

Reply:
Will change in a follow-up pr.